### PR TITLE
doom-bcc-git: init at 2018.01.04

### PR DESCRIPTION
**Pull request by esoeylemez**

### CODE REVIEW

**Comment by Mic92** [comment]
@GrahamcOfBorg  build doom-bcc-git

**Comment by Mic92** [comment]
I suppose for `darwin` we need:

```
makeFlags = [ "CC=cc" ];
```

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -1893,6 +1893,8 @@ with pkgs;
   };
   doomseeker = callPackage ../applications/misc/doomseeker { };
 
+  doom-bcc-git = callPackage ../games/zdoom/bcc-git.nix { };
```
Comment (Mic92) [inline pkgs/top-level/all-packages.nix]: Please remove `-git` suffix here. We use `Unstable` if we have both version of the software.

**Comment by esoeylemez** [comment]
I've added the flag, but can I test this before pushing the commit?

**Comment by esoeylemez** [comment]
At least on x86_64-linux it compiles fine with `CC=cc`.

**Comment by grahamc** [comment]
Once you push it, we can trigger the build again on Darwin via ofborg.

**Comment by grahamc** [comment]
@GrahamcOfBorg eval

**Comment by grahamc** [comment]
@GrahamcOfBorg build doom-bcc-git

**Comment by grahamc** [comment]
_sigh_ I'm generating a lot of spam tonight.

@GrahamcOfBorg build doom-bcc

**Comment by Mic92** [comment]
@GrahamcOfBorg build doom-bcc

**Pull request merged**